v3/setup/splunk.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
@chr0n1x this file is still needed if we don't want to require all of these for every launch. The values set here should be defaults, which the configs could override if needed.